gridfs/gridfs.c

```diff
@@ -550,7 +550,7 @@ PHP_METHOD(MongoGridFS, storeBytes)
  * - upload date
  * - length
  * these fields are only added if the user hasn't defined them. */
-static int setup_file_fields(zval *zfile, char *filename, int length TSRMLS_DC)
+static int setup_file_fields(zval *zfile, char *filename, long length TSRMLS_DC)
```

Comment:
This is passed to `add_assoc_long()`. Also, the value from the `MongoGridFS::storeFile()` code path comes from `setup_file()` and `ftell()`, which returns a `long`. I don't see any reason to switch to `size_t` when both the input and output are already `long`.